recipes/open62541/all/conanfile.py

```diff
@@ -353,18 +354,14 @@ def generate(self):
 
         tc.variables["UA_COMPILE_AS_CXX"] = self.options.cpp_compatible
 
-        # Honor BUILD_SHARED_LIBS from conan_toolchain (see https://github.com/conan-io/conan/issues/11840)
-        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
```

Comment:
Already set by Conan in newer versions

Reply:
Both for conan v1 and conan v2?

Reply:
Ooops, I missread the policy number! This would actually be needed in both versions, Conan does not mess with this one, sorry for the confussion :)